_tutorial_base/800-ci.md

```diff
@@ -18,13 +18,13 @@ So we only have to fire up a shell and run the build!
 
 You do not even need to have gradle installed. There is a script included, `gradlew`, that will download the correct gradle version. You can also install gradle (>= 1.12) and use it. It should go without saying, but you should also have java 8 installed on the command line.
 
-	$ cd /Ws/com.acme.prime
 	$ java -version
 	java version "1.8.0"
+	$ cd /Users/aqute/git/com.acme.prime
```

Comment:
Thanks for fixing the commit comment. But you still did not fix the fact that you moved the `cd` command from the first command in the example to in the middle of the output from the `java -verison` command. 

It used to look like this:

    $ cd /Ws/com.acme.prime
    $ java -version
    java version "1.8.0"
    Java(TM) SE Runtime Environment ...
    $ ./gradlew

but now looks like this:

    $ java -version
    java version "1.8.0"
    $ cd /Users/aqute/git/com.acme.prime
    Java(TM) SE Runtime Environment ...
    $ ./gradlew

See <https://github.com/jconlon/osgi.enroute.site/blob/8060a439d5a472606fcdfe8ec3973da9cb95b0a4/_tutorial_base/800-ci.md#building-outside-the-ide>